devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -188,6 +191,9 @@
     @Parameter(property = "enableRegistryClient")
     private boolean enableRegistryClient;
 
+    @Parameter(property = "configProperties")
+    private String configProperties;
```

Comment:
```suggestion
    @Parameter(property = "config")
    private String config;
```

Since it's dynamic, this makes more sense I guess.